classes/local/store/s3/client.php

```diff
@@ -570,16 +570,18 @@ public function get_expiration_time($now, $expires) {
         if ($expires == 0) {
             $expires = $now + $this->expirationtime;
         }
-        // If it's already expired set it to now.
+        // If it's already expired set it to 1 minute.
         if ($expires < $now) {
-            $expires = $now;
+            $expires = $now + MINSECS;
+        }
+        // If it's greater than a week set it to 1 week.
```

Comment:
I think it's important to say why we are doing this in the comment

Reply:
Added